MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangScheduler.java

```diff
@@ -0,0 +1,160 @@
+package frc.team670.mustanglib.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.DriverStation;
+
+import frc.team670.mustanglib.utils.MustangNotifications;
+import frc.team670.mustanglib.subsystems.MustangSubsystemBase;
+import frc.team670.mustanglib.subsystems.MustangSubsystemBase.HealthState;
+import frc.team670.mustanglib.utils.Logger;
+
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * Responsible for scheduling and running commands, including
+ * MustangCommandBases. Use this instead of the WPILib CommandScheduler.
+ * 
+ * @author ctychen, lakshbhambhani
+ */
+
+public class MustangScheduler {
```

Comment:
same question here - why is this a new file as opposed to a rename/move of the old file?